### PR TITLE
Core-AAM: add tests for sectionheader and sectionfooter

### PR DESCRIPTION
**Pull request by spectranaut**
Change in ARIA: https://github.com/w3c/aria/pull/1931

### CODE REVIEW

File: core-aam/sectionfooter-manual.html
Comment (rahimabdi) [inline core-aam/sectionfooter-manual.html:90]: Minor nit: the `role` and `id` attributes for this `<div>` use single quotes (which is OK so long as single/double quotes are used in matching pairs from my understanding) but double quotes are used elsewhere in this file.

File: core-aam/sectionheader-manual.html
Comment (rahimabdi) [inline core-aam/sectionheader-manual.html:90]: Minor nit: the `role` and `id` attributes for this `<div>` use single quotes (which is OK so long as single/double quotes are used in matching pairs from my understanding) but double quotes are used elsewhere in this file.

**Review by rahimabdi — APPROVED** [review]

File: wai-aria/role/roles.html
Comment (cookiecrook) [inline wai-aria/role/roles.html:107]: Since all implementations have a different mappings (to `generic`), should this be a separate PR from the ATTAcomm tests?

Comment (cookiecrook) [reply wai-aria/role/roles.html:107]: Also, it be in a `.tentative` file until there is a spec change landed and two passing implementations.

**Review by cookiecrook — CHANGES_REQUESTED** [review]
This would result in new failures for Interop, so please keep the WebDriver role verifications out. Adding to a `roles.tentative.html` file is fine, as those don't affect the score.

Even if we agreed on the test additions, there's a process inside the current Interop project together sign off from implementors before making new changes that would affect the score mid-year.

**Comment by spectranaut** [comment]
Ok @cookiecrook -- I moved sectionheader and sectionfooter to a tentative file: https://github.com/web-platform-tests/wpt/pull/46142

If we need sign off from implementers for WPT tests of new futures added to ARIA, we should note that in the ARIA process documents, I'll ping you for a review of that draft.

**Comment by cookiecrook** [comment]
> I moved sectionheader and sectionfooter to a tentative file: https://github.com/web-platform-tests/wpt/pull/46142

Thanks. FYI there is still an accidental white space diff remaining in roles.html.

**Comment by tranjocelyn** [comment]
Is there a PR to update the header and footer (scoped to main/sectioning content) subtests in html-aam/roles-contextual.html and wai-aria/role/contextual-roles.html? 

I have the implementation for sectionheader and sectionfooter ready for Chrome, but it's currently failing those subtests. I'm holding off on merging the code because it would negatively impact our Interop score.

**Comment by spectranaut** [comment]
> Is there a PR to update the header and footer (scoped to main/sectioning content) subtests in html-aam/roles-contextual.html and wai-aria/role/contextual-roles.html? 
> 
> I have the implementation for sectionheader and sectionfooter ready for Chrome, but it's currently failing those subtests. I'm holding off on merging the code because it would negatively impact our Interop score.

Hi Joceyln, if there is a missing test or failing test, maybe make an issue on ARIA with the details so we can update it? This PR just has manual tests to keep track of the API mappings.

File: core-aam/sectionfooter-manual.html
Comment (cookiecrook) [inline core-aam/sectionfooter-manual.html:44]: These are user-presented strings, so two-word role descriptions should include the space. "section footer"

Did you speak to anyone else at Apple about the potential for a new AXSubrole? If not, I'll ask around.

File: core-aam/sectionfooter-manual.html
Comment (cookiecrook) [inline core-aam/sectionfooter-manual.html:74]: I would expect the same (include the space char) is true for _localized_ control types in UIA, but I don't know.

File: core-aam/sectionheader-manual.html
Comment (cookiecrook) [inline core-aam/sectionheader-manual.html:45]: ditto

File: core-aam/sectionheader-manual.html
Comment (cookiecrook) [inline core-aam/sectionheader-manual.html:75]: ditto

File: core-aam/sectionfooter-manual.html
Comment (cookiecrook) [inline core-aam/sectionfooter-manual.html:44]:
```suggestion
                  "section footer"
```

File: core-aam/sectionfooter-manual.html
Comment (cookiecrook) [inline core-aam/sectionfooter-manual.html:38]:
```suggestion
                  "AXSectionFooter"
```

File: core-aam/sectionfooter-manual.html
Comment (cookiecrook) [inline core-aam/sectionfooter-manual.html:74]:
```suggestion
                  "section footer"
```

File: core-aam/sectionheader-manual.html
Comment (cookiecrook) [inline core-aam/sectionheader-manual.html:38]:
```suggestion
                  "AXSectionHeader"
```

File: core-aam/sectionheader-manual.html
Comment (cookiecrook) [inline core-aam/sectionheader-manual.html:44]:
```suggestion
                  "section header"
```

File: core-aam/sectionheader-manual.html
Comment (cookiecrook) [inline core-aam/sectionheader-manual.html:74]:
```suggestion
                  "section header"
```

**Review by cookiecrook — APPROVED** [review]
Approving with the updates from the spec PR

**Review by rahimabdi — COMMENTED** [review]
@cookiecrook has kindly provided comments/suggested feedback.

**Pull request merged**
